src/cpp/utils.cpp

```diff
@@ -0,0 +1,33 @@
+#include "utils.h"
+
+#include <fstream>
+#include <iomanip>
+
+namespace effil {
+
+Logger getLogger() {
+    static std::mutex lock;
+    static const char* logFile = getenv("EFFIL_LOG");
+
+    if (logFile == nullptr) {
```

Comment:
You can store `std::ostream` in Logger and customize it with (ios::rdbuf)[http://www.cplusplus.com/reference/ios/ios/rdbuf/].
Or you can grab reference to `std::cout`.

Reply:
`Logger` already grab stream objects by references